### ISSUE TITLE
Can you provide the training weights for your code?

### ISSUE THREAD

**Issue by 20230307** [aabidk20/KinD-pytorch#1]
Can you provide the training weights for your code?  decom_net_path、illum_net_path、restore_net_path

**Comment by aabidk20** [comment]
Hi, the implementation still has some issues, and the training is not able to achieve the effect of original weights. I have narrowed down the bug to the use of absolute gradients in loss calculations of Decomposition Net, if you like to explore it further.

**Comment by bwzhang163** [comment]
> 嗨，实现上仍然存在一些问题，训练无法达到原始权重的效果。我已经将错误缩小到分解网络损失计算中使用绝对梯度，如果你想进一步探索它。

Can you tell me specifically if there is a problem with the gradient() and gradient_no_abs() functions? Is there a solution to this problem at the moment please?

**Comment by aabidk20** [comment]
@bwzhang163 I couldn't find the root cause back then, as of now, I have dropped it.
